Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_External_-_Generic_v2.yml

```diff
@@ -452,10 +413,10 @@ tasks:
     isautoswitchedtoquietmode: false
   "38":
     id: "38"
-    taskid: 19fff3fe-0e05-4ccf-80ad-c1938b9a7806
+    taskid: bf0fa0db-39d0-4074-8878-c38077334c7a
     type: regular
     task:
-      id: 19fff3fe-0e05-4ccf-80ad-c1938b9a7806
+      id: bf0fa0db-39d0-4074-8878-c38077334c7a
       version: -1
       name: Get IP report from VirusTotal Private API
```

Comment:
This command is deprecated so why don't we remove it? 
and leave the !IP?
Note that it uses a dedicated instance ( which usually it's a bad practice in PBs).

Reply:
Agreed, I don't want to break BC in that case. Some may still be using it. When we release v3 we'll do it for sure